cmd/lncli/main.go

```diff
@@ -2,28 +2,33 @@ package main
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
+
+	"gopkg.in/macaroon-bakery.v1/bakery/checkers"
+	"gopkg.in/macaroon.v1"
```

Comment:
Unfortunately, `macaroon-bakery` actually imports itself, the macaroon package, and other libraries using `gopkg.in`. See, for example, https://github.com/go-macaroon-bakery/macaroon-bakery/blob/v1/bakery/service.go#L14

We can fork these libraries or submit upstream PRs to remove the dependency on `gopkg.in`, but I haven't been able to get it to work without that.